### DISCUSSION TITLE
Not made for empty projects: "Or any other tooling available."

### DISCUSSION THREAD

**Discussion by Sebi2020** [storybookjs/storybook#17630] [Documentation]
Hello,
the documentation states that storybook.js is not made / does not work for / with empty projects. If you look into the documentation it looks like storybook supports the following frameworks:
* Angular (Version xyz?)
* Vue (2 or 3?)
* Ember.js
* React

The last bullet point says "Or any other tooling available". This is a very broad answer if it does not work out-of-the-box in empty projects, so I would be interested in knowing what are this other tools storybook.js supports too or what is needed to make storybook.js work with custom code? For example if I develop web-components without any framework.

**Comment by jonniebigodes** [comment]
@Sebi2020 thanks for reaching out to us with your question we appreciate it. Indeed you're right that is broader as there are so many, for instance, there's a Storybook implementation for [PHP](https://github.com/area17/blast), even a [Rails](https://orbit.love/blog/building-a-component-library-in-rails-with-storybook) implementation. To answer your question if you want to work without any framework and write custom web components, your safest bet would probably be `@storybook/html`. If you're interested check out this [article](https://tobiasljungstrom.net/blog/2021-04-15-web-components-in-storybook/) to get started. 

Hope I was able to provide a bit of context and help you. 

Have a fantastic weekend!

Stay safe
